framework/cmd/ci.go

```diff
@@ -277,3 +295,61 @@ func getColorPrinter(duration time.Duration) *color.Color {
 		return color.New(color.FgRed)
 	}
 }
+
+func printSummary(name string, s *Stat, typ string) {
+	cp50 := getColor(s.P50)
+	cp95 := getColor(s.P95)
+	cp99 := getColor(s.P99)
+	var cpFlaky *color.Color
+	if s.ReRuns > 0 {
+		cpFlaky = color.New(color.FgRed)
+	} else {
+		cpFlaky = color.New(color.FgGreen)
+	}
+	if len(name) > MaxNameLen {
+		name = name[:MaxNameLen]
+	}
+	switch typ {
+	case "jobs":
```

Comment:
nit: define constants?

Reply:
Ok, let's do this next time we'll be iterating on it